spec/csr_spec.rb

```diff
@@ -285,7 +290,7 @@
   it "loads a csr with load_from_file" do
     path = File.dirname(__FILE__) + '/fixtures/csr1.pem'
     csr = R509::CSR.load_from_file path
-    expect(csr.message_digest.name).to eq('sha1')
+    expect(csr.message_digest.name).to eq('sha256')
```

Comment:
Yes, I thought I'd change the _default_ to sha256 as well as there were no other. I couldn't consistently do this, `cert1.pem` for example I couldn't change without replacing the cert.